.gitignore

```diff
@@ -13,3 +13,5 @@ web-report/
 # macOS
 .DS_Store
 
+# Ignore folder named `coverage` 
+coverage/
```

Comment:
@EmilyLalonde For some reason, this isn't working and we're still getting all of the coverage files pushed up. 

Can we try a `/coverage` instead?